app/helpers/parent_interface/consent_forms_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ParentInterface::ConsentFormsHelper
```

Comment:
I can't see where this helpers are used, do we need them?

Reply:
Ah, this was abandoned, thought I had removed it. 👍